.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         toolchain: [ "gcc", "clang"]
         protocol: ["current", "next"]
-        scenario: ["", "--check-test-tx-meta"]
+        scenario: ["", "--check-test-tx-meta", "--disable-tests"]
```

Comment:
This does not do what you intend: this is an argument to `ci-build.sh` to tell it to just build and not run tests, it still builds without `disable-tests` passed to `configure`.

Also, you don't want caches to be trashed by the new build flavor (see cache key), so we may need a `flavor` dimension instead (that gets skipped in scenarios other than "")